azurerm/internal/services/loganalytics/parse/log_analytics_workspace.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 type LogAnalyticsWorkspaceId struct {
+	Subscription  string
```

Comment:
can we make this `SubscriptionId` here?

Reply:
Instead, I removed that field as it is not used anywhere and we already has implement the IDFormatter for this type.